rustls/Cargo.toml

```diff
@@ -14,6 +14,65 @@ autotests = false
 exclude = ["src/testdata", "tests/**"]
 build = "build.rs"
 
+[features]
+default = ["aws_lc_rs", "logging", "std", "tls12"]
+
+## Enable some more advanced connection support APIs.
+std = ["webpki/std", "pki-types/std", "once_cell/std"]
```

Comment:
I think this documentation needs to be more specific, would appreciate any pointers to help improve this.